pkg/virt-controller/watch/vm/vm_test.go

```diff
@@ -7141,6 +7142,57 @@ var _ = Describe("VirtualMachine", func() {
 		)
 	})
 
+	Context("startVMI", func() {
```

Comment:
I don't think we need this Context. We can put the test in the `One valid VirtualMachine controller given` context, saving a lot of code (setup of controller, informers, etc..). WDYT?